build-tests/heft-sass-test/config/heft.json

```diff
@@ -36,16 +36,27 @@
    * The list of Heft plugins to be loaded.
    */
   "heftPlugins": [
-    // {
-    //  /**
-    //   * The path to the plugin package.
-    //   */
-    //  "plugin": "path/to/my-plugin",
-    //
-    //  /**
-    //   * An optional object that provides additional settings that may be defined by the plugin.
-    //   */
-    //  // "options": { }
-    // }
+    {
+      /**
+       * The path to the plugin package.
+       */
+      "plugin": "@rushstack/heft-webpack4-plugin/lib/WebpackPlugin"
+
+      /**
+       * An optional object that provides additional settings that may be defined by the plugin.
+       */
+      // "options": { }
+    },
+    {
+      /**
+       * The path to the plugin package.
+       */
+      "plugin": "@rushstack/heft-webpack4-plugin/lib/BasicConfigureWebpackPlugin"
```

Comment:
Is there a way that the various "configure" plugins could include the "webpack" support with them?

It's slightly awkward (i.e. more explaining in the docs) that people have to load two plugins to accomplish one goal.

Reply:
Updated this design.